go.work.sum

```diff
@@ -0,0 +1,711 @@
+cloud.google.com/go v0.110.2/go.mod h1:k04UEeEtb6ZBRTv3dZz4CeJC3jKGxyhl0sAiVVquxiw=
```

Comment:
Could you remove this file?

Reply:
removed in https://github.com/equinix/terraform-provider-equinix/pull/406/commits/42f2da654a27f8c93a6220fc4a2fa8bb8b0107f1